bgpd/bgp_routemap.c

```diff
@@ -6082,6 +6084,23 @@ DEFUN_YANG (no_match_ipv6_next_hop,
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+ALIAS_HIDDEN (match_ipv6_next_hop,
```

Comment:
Maybe we should add a deprecation cycle for this behavior?

Reply:
Sure, I can add a CPP_NOTICE. But first of all we need to decide which way we'll go – mine or what Donald suggested :)